### ISSUE TITLE
Missing Dictionary of Valid Values for the `platform-family` and the `platform-name` `inspec.yml` Entries

### ISSUE THREAD

**Issue by christopher-m-hosmer** [inspec/inspec#6855]
# Describe the problem
The InSpec __Profiles__ documentation does not provide a complete dictionary of valid values for either the `platform-family` or the `platform-name` entries.  The documentation only provides a limited set of valid examples.
<https://github.com/inspec/inspec/blob/main/docs-chef-io/content/inspec/profiles.md>

## Possible Solution
Include a dictionary of valid values either on the page or a link to the dictionary.

**Comment by ahasunos** [comment]
Getting addressed here: #6857 
```
❯ inspec schema platforms | jq
[
  {
    "name": "aix",
    "families": [
      "aix",
      "unix",
      "os"
    ]
  },
  {
    "name": "alpine",
    "families": [
      "linux",
      "unix",
      "os"
    ]
  },
  {
    "name": "amazon",
    "families": [
      "redhat",
      "linux",
      "unix",
      "os"
    ]
  },
  {
    "name": "arch",
    "families": [
      "linux",
      "unix",
      "os"
    ]
  },
  {
    "name": "arista_eos",
    "families": [
      "arista_eos",
      "linux",
      "unix",
      "os",
      "os"
    ]
  },
  {
    "name": "arista_eos_bash",
    "families": [
      "arista_eos",
      "linux",
      "unix",
      "os",
      "os"
    ]
  },
  {
    "name": "aws",
    "families": [
      "cloud",
      "api"
    ]
  },

.......... snip .......... 

]
```

**Comment by clintoncwolfe** [comment]
We've merged in a docs update. Thanks!
